src/draw/nxp/pxp/lv_gpu_nxp_pxp_osa.c

```diff
@@ -141,20 +142,25 @@ static lv_res_t _lv_gpu_nxp_pxp_interrupt_init(void)
         return LV_RES_INV;
 
     NVIC_SetPriority(LV_GPU_NXP_PXP_IRQ_ID, configLIBRARY_MAX_SYSCALL_INTERRUPT_PRIORITY + 1);
+    NVIC_EnableIRQ(LV_GPU_NXP_PXP_IRQ_ID);
+
+#elif defined(__ZEPHYR__)
+    IRQ_DIRECT_CONNECT(LV_GPU_NXP_PXP_IRQ_ID, CONFIG_DMA_INIT_PRIORITY, PXP_IRQHandler, 0);
+    irq_enable(LV_GPU_NXP_PXP_IRQ_ID);
 #endif
-    s_pxpIdle = true;
 
-    NVIC_EnableIRQ(LV_GPU_NXP_PXP_IRQ_ID);
+    s_pxpIdle = true;
 
     return LV_RES_OK;
 }
 
 static void _lv_gpu_nxp_pxp_interrupt_deinit(void)
 {
-    NVIC_DisableIRQ(LV_GPU_NXP_PXP_IRQ_ID);
```

Comment:
Do we have other OS in use? Maybe add it to `#else ` is better for backward compatibility.

Reply:
Good point, did not consider that this changes behaviour for other OSes

Reply:
Applied your suggestion.